hack/jenkins/test-cmd-dockerized.sh

```diff
@@ -19,20 +19,11 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
-# Runs test-cmd,
-# producing JUnit-style XML test
-# reports in ${WORKSPACE}/artifacts. This script is intended to be run from
-# kubekins-test container with a kubernetes repo mapped in. See
-# k8s.io/test-infra/scenarios/kubernetes_verify.py
+# Runs test-cmd, intended to be run in prow.k8s.io
+set -x;
 
-export PATH=${GOPATH}/bin:${PWD}/third_party/etcd:/usr/local/go/bin:${PATH}
```

Comment:
we don't need to do this, just `source ./hack/install-etcd.sh`, `kube::etcd::install` will update PATH, de-duping the PATH computation

Reply:
and the gopath bin things?

Reply:
The CI image already has PATH setup reasonably aside from installing etcd, which is the correct place to handle this. We handle etcd path below.

You can also seem that the CI jobs pass after this change: 
https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/130760/pull-kubernetes-integration/1900251659217408000
https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/130760/pull-kubernetes-cmd/1900251658978332672

Reply:
When Kubernetes installs a tool into GOPATH/bin, it does it inside of a script, and it does it in a temporary GOPATH under _output or similar.

the install etcd script is an outlier, it doesn't install from source though, and it's old